### PR TITLE
Fedora 40, install ruff from distro packages

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
- [x] rebuild tasks container when landing: https://github.com/cockpit-project/cockpituous/actions/runs/8831813342

### CODE REVIEW

File: tasks/container/Containerfile [hunk 1]
```diff
@@ -50,12 +50,14 @@ RUN dnf -y update && \
         python3-pytest-asyncio \
         python3-pytest-timeout \
         python3-pyyaml \
+        python3-types-pyyaml.noarch \
```
Comment (martinpitt) [inline tasks/container/Containerfile:53]: The .noarch suffix is weird here, let's please drop that.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Cool, I wanted to do the same thing this morning :grin:

File: tasks/container/Containerfile (hunk 1 above)
Comment (allisonkarlitskaya) [reply tasks/container/Containerfile:53]: Copy-paste error.

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**